matching-game/src/pages/index.tsx

```diff
@@ -28,9 +30,11 @@ export const Home = (): JSX.Element => {
 
     if (clickedCard.emoji === emoji) {
       setNumberOfMatches(numberOfMatches + 1)
+      setNumberOfAttempts(numberOfAttempts + 1)
     } else {
       flipCard(index, faceDown[index])
       flipCard(clickedCard.index, true)
+      setNumberOfAttempts(numberOfAttempts + 1)
     }
 
     setClickedCard({ emoji: '', index: -1 })
```

Comment:
Small nitpick: You can actually put `setNumberOfAttempts` down on line 39, since you have to do it regardless of the if statement. That way you don't have to write it in each part of the `if/else`